### PR TITLE
BrenKt-SK-1373-WEB-Domain-DNS-zone-edit

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md [hunk 1]
```diff
@@ -249,27 +249,44 @@ Vous pouvez supprimer plusieurs entrées en une seule fois en les cochant depuis
 
 #### Réinitialiser la zone DNS
 
-Réinitialiser votre zone DNS vous permet de :
+Réinitialiser votre zone DNS permet de revenir à une configuration minimale, avec les entrées OVHcloud par défaut ou celles de vos services. Vous pouvez également pointer votre nom de domaines vers des services d'hébergement Web et e-mail personnalisés .
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md]:
```suggestion
Réinitialiser votre zone DNS permet de revenir à une configuration minimale, avec les entrées OVHcloud par défaut ou celles de vos services. Vous pouvez également pointer votre nom de domaine vers des services d'hébergement Web et e-mail personnalisés .
```

File: pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md [hunk 2]
```diff
@@ -249,27 +249,44 @@ Vous pouvez supprimer plusieurs entrées en une seule fois en les cochant depuis
...
+**Étape 2**
+
+Quel que soit votre choix à l'étape 1, il est nécessaire de définir une réponse lorsqu'on interroge votre nom de domaine pour éviter une réponse DNS en erreur.
+
+Sélectionnez les deux options en cliquant sur les onglets suivants.
+
+> [!tabs]
+> **Adresse IP de votre hébergement**
+>> - `Redirection`: votre nom de domaine pointera vers le serveur de redirection OVHcloud permettant d'afficher une page d'accueil OVHcloud et ainsi éviter une erreur DNS.<br>
+>> - `Hébergement Web OVHcloud`: Votre nom de domaine pointera vers l'adresse IP de l'hébergement Web associé à nom de domaine <br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md]:
```suggestion
>> - `Hébergement Web OVHcloud`: votre nom de domaine pointera vers l'adresse IP de l'hébergement Web associé au nom de domaine <br>
```

File: pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md [hunk 3]
```diff
@@ -249,27 +249,44 @@ Vous pouvez supprimer plusieurs entrées en une seule fois en les cochant depuis
...
+>> - `Personnalisé`: définissez la valeur IPv4 ([enregistrement A](#pointer-records)) de l'hébergement Web que vous souhaitez pointer. <br><br>
+>> ![dnszone](images/dns-zone-reset-01.png){.thumbnail}
+>>
+> **Adresse de votre serveur mail**
+>> - `Redirection`: Votre nom de domaine pointera vers les serveurs redirections e-mails. Ce choix. Il est particulièrement utile si vous n'avez aucune offre e-mail mais que vous souhaitez renvoyer les e-mails vers une ou des adresses e-mails hors de votre nom de domaine.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md]:
```suggestion
>> - `Redirection`: votre nom de domaine pointera vers les serveurs de redirections e-mail. Ce choix est particulièrement utile si vous n'avez aucune offre e-mail mais que vous souhaitez renvoyer les e-mails vers une ou des adresses e-mails en dehors de votre nom de domaine.
```

**Review by OVH-Team-Guides — COMMENTED** [review]
Date update also needed

File: pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md (hunk 1 above)
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md]:
```suggestion
Réinitialiser votre zone DNS permet de revenir à une configuration minimale, avec les entrées OVHcloud par défaut ou celles de vos services. Vous pouvez également faire pointer votre nom de domaine vers des services d'hébergement Web et d'e-mail personnalisés .
```

File: pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md [hunk 4]
```diff
@@ -249,27 +249,44 @@ Vous pouvez supprimer plusieurs entrées en une seule fois en les cochant depuis
...
 #### Réinitialiser la zone DNS
 
-Réinitialiser votre zone DNS vous permet de :
+Réinitialiser votre zone DNS permet de revenir à une configuration minimale, avec les entrées OVHcloud par défaut ou celles de vos services. Vous pouvez également pointer votre nom de domaines vers des services d'hébergement Web et e-mail personnalisés .
 
-- soit revenir à une configuration minimale avec les entrées OVHcloud par défaut.
-- soit revenir à une zone DNS vide (à l'exception des champs NS), pour définir une configuration manuelle ultérieure.
+> [!alert]
+>
+> Avant de réinitialiser votre zone DNS, assurez-vous que votre nom de domaine n'est pas attaché à des services en cours d'utilisation, tels qu'un site web ou des adresses e-mail.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md:256]:
```suggestion
> Avant de réinitialiser votre zone DNS, vérifiez que votre nom de domaine n'est pas attaché à des services en cours d'utilisation, tels qu'un site web ou des adresses e-mail.
```

File: pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md [hunk 5]
```diff
@@ -249,27 +249,44 @@ Vous pouvez supprimer plusieurs entrées en une seule fois en les cochant depuis
...
 
-Vous avez le choix entre :
+**Étape 1**
 
-- `Oui, je veux réinitialiser ma zone DNS avec les entrées minimales`. Cette option vous permet de diriger votre nom de domaine et votre service e-mail vers :
-    - l'un de vos services Web Cloud disponibles dans votre espace client OVHcloud.
-    - le service de redirection OVHcloud, accessible via l'onglet `Redirection`{.action} de votre nom de domaine dans les sections `Noms de domaine`{.action} et `Emails`{.action}.
-    - la fonction `Personnalisé`. Saisissez le champ `A` et/ou `MX` de votre choix.
-- `Non, mais je veux réinitialiser ma zone DNS`. Votre zone DNS sera alors vide, à l'exception des entrées NS qui seront automatiquement attachées aux serveurs DNS OVHcloud de votre zone DNS.
+Répondez à la question `Voulez-vous activer les entrées minimales lors de la réinitialisation de votre zone DNS ?`. Le principe de définir des entrées minimales dans une zone DNS est d'éviter qu'une requête vers le nom de domaine n'aboutisse pas sur une erreur.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md]:
```suggestion
Répondez à la question `Voulez-vous activer les entrées minimales lors de la réinitialisation de votre zone DNS ?`. Définir des entrées minimales dans une zone DNS permet d'éviter qu'une requête vers le nom de domaine aboutisse sur une erreur.
```

File: pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md [hunk 6]
```diff
@@ -249,27 +249,44 @@ Vous pouvez supprimer plusieurs entrées en une seule fois en les cochant depuis
...
+
+**Étape 2**
+
+Quel que soit votre choix à l'étape 1, il est nécessaire de définir une réponse lorsqu'on interroge votre nom de domaine pour éviter une réponse DNS en erreur.
+
+Sélectionnez les deux options en cliquant sur les onglets suivants.
+
+> [!tabs]
+> **Adresse IP de votre hébergement**
+>> - `Redirection`: votre nom de domaine pointera vers le serveur de redirection OVHcloud permettant d'afficher une page d'accueil OVHcloud et ainsi éviter une erreur DNS.<br>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md]:
```suggestion
>> - `Redirection`: votre nom de domaine pointera vers le serveur de redirection OVHcloud. Cela permet d'afficher une page d'accueil OVHcloud et ainsi éviter une erreur DNS.<br>
```

File: pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md (hunk 2 above)
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md]:
```suggestion
>> - `Hébergement Web OVHcloud`: Votre nom de domaine pointera vers l'adresse IP de l'hébergement Web associé à votre nom de domaine <br>
```

File: pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md (hunk 3 above)
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.fr-fr.md]:
```suggestion
>> - `Redirection`: Votre nom de domaine pointera vers les serveurs de redirections e-mails. Ce choix est particulièrement utile si vous n'avez aucune offre e-mail mais que vous souhaitez renvoyer les e-mails vers une ou plusieurs adresses e-mails hors de votre nom de domaine.
```

**Comment by BrenKt** [comment]
translation check needed for all languages

File: pages/web_cloud/domains/dns_zone_edit/guide.es-es.md [hunk 7]
```diff
@@ -261,27 +262,43 @@ Puede borrar varias entradas de una vez marcándolas desde la parte izquierda de
...
-> [!primary]
->
-> Antes de restaurar la zona DNS, asegúrese de que el dominio no está asociado a servicios en uso, como un sitio web o direcciones de correo.
->
+- `Sí, quiero restaurar mi zona DNS con los registros mínimos`
+- `No, pero quiero restaurar mi zona DNS`
+
+**Etapa 2**
+
+Independientemente de cuál sea su elección en el paso 1, es necesario definir una respuesta cuando se pregunta a su nombre de dominio para evitar una respuesta DNS en error.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.es-es.md]:
```suggestion
Independientemente de cuál sea su elección en el etapa 1, es necesario definir una respuesta cuando se pregunta a su nombre de dominio para evitar una respuesta DNS en error.
```

File: pages/web_cloud/domains/dns_zone_edit/guide.it-it.md [hunk 8]
```diff
@@ -187,8 +187,9 @@ Selezionare il record desiderato facendo clic su ognuna delle schede seguenti.
 >>
 >> > [!warning]
 >> >
->> > Se utilizzi un certificato SSL Let's Encrypt su un hosting condiviso OVHcloud e utilizzi un record CAA, quest'ultimo impedirà la rigenerazione del certificato SSL Let's Encrypt.
+>> > Se configuri una voce CAD per un dominio, questa configurazione verrà applicata anche a **tutti i sottodomini** dello stesso dominio.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.it-it.md]:
```suggestion
>> > Se configuri una voce CAA per un dominio, questa configurazione verrà applicata anche a **tutti i sottodomini** dello stesso dominio.
```

File: pages/web_cloud/domains/dns_zone_edit/guide.pt-pt.md [hunk 9]
```diff
@@ -254,27 +255,43 @@ Pode eliminar várias entradas de uma só vez, selecionando-as a partir da parte
...
 
-Pode escolher entre:
+**Etapa 1**
 
-- `Sim, quero reiniciar a minha zona DNS com o número mínimo` de entradas. Esta opção permite-lhe dirigir o seu nome de domínio e o seu serviço de e-mail para:
-    - um dos serviços Web Cloud disponíveis na Área de Cliente OVHcloud.
-    - o serviço de reencaminhamento OVHcloud, acessível através do separador `Reencaminhamento`{.action} do seu nome de domínio nas secções `Nomes de domínio`{.action} e `Emails`{.action}.
-    - a função `Personalizada`. Introduza o campo `A` e/ou `MX` à sua escolha.
-- `Não, mas quero reiniciar a minha zona DNS`. A sua zona DNS estará vazia, com exceção das entradas NS que serão automaticamente associadas aos servidores DNS da OVHcloud da sua zona DNS.
+Responda à pergunta "Deseja ativar o número mínimo de entradas quando a zona DNS for reiniciada?" Definir entradas mínimas numa zona DNS permite evitar que um pedido para o domínio não acarrete um erro.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.pt-pt.md]:
```suggestion
Responda à pergunta `Deseja ativar o número mínimo de entradas quando a zona DNS for reiniciada?` Definir entradas mínimas numa zona DNS permite evitar que um pedido para o domínio não acarrete um erro.
```

File: pages/web_cloud/domains/dns_zone_edit/guide.pt-pt.md [hunk 10]
```diff
@@ -254,27 +255,43 @@ Pode eliminar várias entradas de uma só vez, selecionando-as a partir da parte
...
 
-- ou voltar a uma configuração mínima com as entradas OVHcloud predefinidas.
-- voltar a uma zona DNS vazia (exceto os campos NS), para definir uma configuração manual posterior.
+> [!alert]
+>
+> Antes de reiniciar a zona DNS, certifique-se de que o seu domínio não está associado a serviços em curso de utilização, tais como um website ou endereços de e-mail.
+>
 
-No separador `Zona DNS`{.action}, clique em `Reiniciar a minha zona DNS`{.action} e siga os passos que aparecem.
+No separador "Zona DNS`{.action}", clique em "Reiniciar a minha zona DNS`{.action}" e siga os 2 passos que aparecem.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/dns_zone_edit/guide.pt-pt.md]:
```suggestion
No separador `Zona DNS`{.action}, clique em `Reiniciar a minha zona DNS`{.action} e siga os 2 passos que aparecem.
```

**Review by Kazarus4 — APPROVED** [review]

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**